core/modules/widgets/action-listops.js

```diff
@@ -3,87 +3,89 @@ title: $:/core/modules/widgets/action-listops.js
 type: application/javascript
 module-type: widget
 
-Action widget to apply list operations to any tiddler field (defaults to the 'tags' field of the current tiddler)
+Action widget to apply list operations to any tiddler field (defaults to the 'list' field of the current tiddler)
 
 \*/
 (function () {
 
-  /*jslint node: true, browser: true */
-  /*global $tw: false */
-  "use strict";
+    /*jslint node: true, browser: true */
+    /*global $tw: false */
+    "use strict";
 
-  var Widget = require("$:/core/modules/widgets/widget.js").widget;
+    var Widget = require("$:/core/modules/widgets/widget.js")
+        .widget;
 
-  var ActionListopsWidget = function (parseTreeNode, options) {
-    this.initialise(parseTreeNode, options);
-  };
+    var ActionListopsWidget = function (parseTreeNode, options) {
+        this.initialise(parseTreeNode, options);
```

Comment:
>-- my problem is working with Github.

yea. tabs are 8 characters wide :/ ... and to fix it, one needs to install a plugin from the guy, that was responsible, for the settings removal. [What a joke](https://addons.mozilla.org/en-US/firefox/addon/custom-tab-width/reviews/766795/)